core/domain/exp_jobs_one_off_test.py

```diff
@@ -1133,16 +1127,16 @@ def test_migration_job_audit_success(self):
         }
 
         self.create_exploration_with_states_schema_version(
-            36,
+            37,
```

Comment:
Why do you need to change all these? The idea was to fix the start and end versions so that this doesn't need to be changed each time someone wants to do a migration.

Reply:
I see. I've reverted these changes now.